### PR TITLE
Files table

### PR DESCRIPTION
**Pull request by itbf9**
This PR includes:
- New files datasource
- New files-table component
- Change logo in dark mode

### CODE REVIEW

File: src/app/core/_models/file.model.ts [hunk 1]
```diff
@@ -1,26 +1,52 @@
-import { AccessGroup } from './access-group.model'
+import { AccessGroup } from './access-group.model';
 
+export enum FileType {
```
Comment (xbenyx) [inline src/app/core/_models/file.model.ts:3]: Im using something similar in core/_constants/files.config

File: src/app/files/files-edit/files-edit.component.html [hunk 2]
```diff
@@ -3,57 +3,57 @@
 <grid-main [class]="'width:100%;'" [centered]="true">
```
Comment (xbenyx) [inline src/app/files/files-edit/files-edit.component.html:3]: are you adding this component back? It doesnt exist

**Review by xbenyx — CHANGES_REQUESTED** [review]
Nice, just 2 things to add.

File: src/app/core/_models/file.model.ts (hunk 1 above)
Comment (itbf9) [reply src/app/core/_models/file.model.ts:3]: Yes I know, this enum will replace that one was my thought.

File: src/app/files/files-edit/files-edit.component.html (hunk 2 above)
Comment (itbf9) [reply src/app/files/files-edit/files-edit.component.html:3]: What do you mean? It exists :) I think I just saved this one and it was formatted, I haven't changed anything.

Comment (xbenyx) [reply src/app/files/files-edit/files-edit.component.html:3]: Ohh it does exist lol nevermind.

File: src/app/core/_models/file.model.ts (hunk 1 above)
Comment (xbenyx) [reply src/app/core/_models/file.model.ts:3]: Yes, that sounds perfect.

**Review by xbenyx — APPROVED** [review]

**Pull request merged**